compiler/src/dotty/tools/dotc/core/TypeErasure.scala

```diff
@@ -320,6 +320,7 @@ object TypeErasure {
         case _ => false
       }
     case tp: TypeParamRef => false
+    case tp: TypeBounds => false
```

Comment:
Would `hasStableErasure(tp.lo) && hasStableErasure(tp.hi)` be incorrect?

Reply:
No, because we fall into this case also via TypeProxy.